README.md

```diff
@@ -405,65 +426,28 @@ parameter_defaults:
     rgw_swift_versioning_enabled: true
 ```
 
-Once you customize the above template/examples for your environment you can create/update configmaps for both the 'tripleo-deploy-config-custom' and 'tripleo-net-config' ConfigMaps by using these example commands on the files containing each respective configmap type (one directory for each type of configmap):
-
-```bash
-# create the configmap for tripleo-deploy-config-custom
-oc create configmap -n openstack tripleo-deploy-config-custom --from-file=tripleo-deploy-config-custom/ --dry-run -o yaml | oc apply -f -
-
-# create the configmap for netconfig
-oc create configmap -n openstack tripleo-net-config --from-file=net-config/ --dry-run -o yaml | oc apply -f -
-```
+Once you customize the above template/examples for your environment, create/update configmaps like explained in `Deploying OpenStack once you have the OSP Director Operator installed`
 
 ## Render playbooks and apply them
 
-* In openstackclient create the roles file for the overcloud, like Controller + ComputeHCI
+* Define an OpenStackPlaybookGenerator to generate ansible playbooks for the OSP cluster deployment as in `Deploying OpenStack once you have the OSP Director Operator installed` and specify the roles generated roles file.
 
-```bash
-unset OS_CLOUD
-openstack overcloud roles generate -o /home/cloud-admin/roles_data_hci.yaml Controller ComputeHCI
-```
-
-* create the tripleo-deploy-custom.sh deployment script and update to use the custom roles file
+## Run the software deployment
 
-
-```bash
-cd
-cp tripleo-deploy.sh tripleo-deploy-custom.sh
-sed -i 's/ROLESFILE/~\/roles_data_hci.yaml/' tripleo-deploy-custom.sh
-```
-
-* add the `ceph-ansible.yaml` environment file and to the `openstack tripleo deploy` command
-
-```bash
-     -e /usr/share/openstack-tripleo-heat-templates/environments/docker-ha.yaml \
-+    -e /usr/share/openstack-tripleo-heat-templates/environments/ceph-ansible/ceph-ansible.yaml \
-     -e ~/config/deployed-server-map.yaml \
-```
-
-* for now add skipping opendev-validation-ceph when run ansible playbooks
-
-```bash
-   time ansible-playbook -i inventory.yaml \
-     --private-key /home/cloud-admin/.ssh/id_rsa \
-+    --skip-tags opendev-validation-ceph \
-     --become deploy_steps_playbook.yaml
-```
-
-* render the ansible playbooks
-
-```bash
-./tripleo-deploy-custom.sh -r
-```
+* In the openstackclient pod
 
 * Install the pre-requisites on overcloud systems for ceph-ansible
 
 ```bash
-ansible -i tripleo-deploy/overcloud-ansible/inventory.yaml overcloud -a "sudo sudo dnf -y install python3 lvm2"
+cd /home/cloud-admin/ansible
+ansible -i inventory.yaml overcloud -a "sudo sudo dnf -y install python3 lvm2"
```

Comment:
Is the double `sudo sudo` needed here?

Reply:
no, definitely not. fixed in latest commit